docs/SETTINGS.md

```diff
@@ -0,0 +1,195 @@
+
+## Moderation options
+
+---
+
+### Interaction limits
+
+Default settings should be alright here and can also be configured across the entire organization 
+in the [organization settings](https://github.com/organizations/strangelove-ventures/settings/interaction_limits).
+
+### Code review limits
+
+Default settings should be alright here.
+
+### Reported content
+
+Default settings should be alright here.
+
+## Branches
+
+---
+
+A new branch protection rule should be created for the `main` branch. This rule should have the following settings:
+
+- Enable require a pull request before merging
+- Enable require approvals with at least 1 approval needed before merging
+- Enable require review from Code Owners
+- Enable require status checks to pass before merging
+  - TODO: Fill out this section with all status checks that should be required
+  - This will have a list of all CI actions that MUST pass in order for a PR to be merged.
+  - A conversation should be had with the team to determine what these checks should be before accepting this doc.
+- Enable require branches to be up-to-date before merging
+- Enable do not allow bypassing the above settings
+
+Additional branch protection rules should be created for branches that are currently being maintained, used for releases,
+and/or other important branches that should not be merged into without proper review and restrictions in place. 
+
+## Tags  
+
+---
+
+Default settings should be alright here. 
+
+## Rules  
+
+---
+
+### Rulesets
+
+Default settings should be alright here.
+
+### Insights 
+
+Default settings should be alright here.
+
+## Actions  
+
+---
+
+### General
+
+#### Actions permissions
+
+- Allow all actions and reusable workflows
+
+#### Artifact and log retention 
+
+- Default value of 90 days should be alright here
+
+#### Fork pull request workflows from outside collaborators
+
+- Require approval for first-time contributors
+
+#### Workflow permissions
+
+- Read and write permissions
+- Allow GitHub Actions to create and approve pull requests
+
+### Runners
+
+Default settings should be alright here.
+
+
+## Webhooks  
+
+---
+
+Default settings should be alright here.
+
+## Copilot  
+
+---
+
+Default settings should be alright here.
+
+## Environments  
+
+---
+
+Default settings should be alright here.
+
+## Codespaces  
+
+---
+
+Default settings should be alright here.
+
+## Pages  
+
+---
+
+Default settings should be alright here.
+
+## Custom Properties 
+
+---
+
+Default settings should be alright here.
+
+## Code security and analysis 
+
+---
+
+- Enable Private vulnerability reporting
+- Enable dependency graph
+
+### Dependabot
+
+- Enable Dependabot alerts
+- Enable Dependabot security updates
+- Enable Dependabot version updates
+
+### Code scanning
+
+- Enable CodeQL analysis
+
+### Secret scanning
+
+- Currently, secret scanning is disabled but can be enabled if needed.
```

Comment:
Any reason not to enable it? We've certainly been saved a few times ourselves by automated scanning for secrets.

Reply:
I don't have any reason to not enable it. I had just looked at a few of our repos and never saw it being used. I'll enable it and change this section

Reply:
added 7e275bd19e76fc82cea4e8e6495d4b0cb4ca7375